code/modules/research/xenoarchaeology/artifact/effect.dm

```diff
@@ -87,13 +87,13 @@ proc/GetAnomalySusceptibility(var/mob/living/carbon/human/H)
 	if(!H || !istype(H))
 		return 1
 
-	var/protected = 0
-
 	//anomaly suits give best protection, but excavation suits are almost as good
 	if(istype(H.back,/obj/item/weapon/rig/hazmat))
 		var/obj/item/weapon/rig/hazmat/rig = H.back
 		if(rig.suit_is_deployed() && !rig.offline)
-			protected += 1
+			return 1 //<- Maximum level of protection achieved.
```

Comment:
This should be 0.

0 is full protection, so ye.

Reply:
Whups, you're right, I'm stoopid ç_ç Corrected soon!